src/main/java/com/dawn/jat/illuminati/post/repository/PostRepository.java

```diff
@@ -1,7 +1,9 @@
 package com.dawn.jat.illuminati.post.repository;
 
 import com.dawn.jat.illuminati.post.entity.PostEntity;
+
```

Comment:
is this intentional for style check or it's just the Intelij auto add

Reply:
Updated

Reply:
This is just my intention. Revert the file